src/aspire/source/image.py

```diff
@@ -803,6 +831,63 @@ def save_images(
                 im.save(mrcs_filepath, overwrite=overwrite)
 
 
+class RemappedSource(ImageSource):
```

Comment:
How about `SlicedSource` or `SubsetSource`? `Remapped` is a bit abstract. Or do `Sliced` or `Subset` indicate that we wouldn't support an arbitrary index map?

Reply:
I thought about those and went with `Remapped` because we might just be reordering or even copying... Probably `MappedSource` would have been better, lol. but of the two new suggestions, I prefer `SubsetSource`. Let's settle on one today and I'll finish this up so Josh and move on with the Oriented Source.

Reply:
:+1: